.github/workflows/ci.yml

```diff
@@ -29,45 +29,135 @@ jobs:
             cppyy: Off
             extra_cmake_options: '-DLLVM_USE_LINKER=lld'
 
+          - name: ubu22-gcc12-clang-repl-17-cppyy
+            os: ubuntu-latest
+            compiler: gcc-12
+            clang-runtime: '17'
+            cling: Off
+            cppyy: On
+            extra_cmake_options: '-DLLVM_USE_LINKER=lld'
+
           - name: ubu22-gcc9-clang-repl-16
-            os: ubuntu-22.04
+            os: ubuntu-latest
```

Comment:
If not explicitly required ubuntu-latest, it is better to keep the specific version. Makes the matrix more stable for github changes.

Reply:
At the time the ci had both ubuntu-22.04 and ubuntu-latest . I didn't know which one to pick. I should have realised from the name. I've now made this change.